src/view/com/lightbox/ImageViewing/components/ImageItem/ImageItem.ios.tsx

```diff
@@ -275,4 +277,37 @@ const getImageStyles = (
   }
 }
 
+const getImageTransform = (
```

Comment:
These two are copypasta from `utils.js`. I'll get back to this file later but tried not to change anything on iOS in this PR.